src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt

```diff
@@ -186,7 +186,8 @@ interface VisitRepository : JpaRepository<Visit, Long>, JpaSpecificationExecutor
       "(:#{#visitStatusList} is null OR v.visitStatus in :visitStatusList) AND " +
       "(:#{#visitRestrictions} is null OR v.visitRestriction in :visitRestrictions) AND " +
       "v.sessionSlot.slotDate >= :fromDate  AND " +
-      "v.sessionSlot.slotDate <= :toDate " +
+      "v.sessionSlot.slotDate <= :toDate AND " +
+      "v.prison.code = :prisonCode " +
```

Comment:
add prison code parameter to SQL query to ensure visits are only being returned for that prison